src/@core/functions.ts

```diff
@@ -34,14 +34,32 @@ export const toArrayId: ToArrayId = (id: string) => Number(id) - 1;
 // to give useParam id a number value and decrease by 1 to match an array index
 export const audioPlayFunc: AudioFunc = (publicURL, src, mode) => {
   const sound = new Audio(`${publicURL}/${src}`);
```

Comment:
Might we should use one audio element for all function calls instead of creatimgn each time?